frontend/ssvg.ts

```diff
@@ -159,7 +163,14 @@ export default class SSVG {
                 if(this.renderer.updatePropertiesFromQueue) {
                     this.renderer.updatePropertiesFromQueue(queue);
                 }
+
+                if(Object.keys(queue).length === 0) {
```

Comment:
Why is this block of code needed?

Reply:
It's basically the same if-condition as used for the worker updating (line 154-158). IIRC this ensures that `this.renderer.draw()` as well as `this.logDrawn()` are only called when something changed and not on every frame.

Reply:
I still don't understand it though. I do get that we use the same code for the worker, but the worker uses a different scheduling mechanism than the same thread method. As you can see in lines 70-80, for the version without the worker, updateCanvas is already continuously being called for every frame.

As far as I can see, what you are doing here is changing the scheduling mechanism for the non-worker method of ssvg. Is this actually related to what this pull request is trying to do? How is it related?